http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -63,14 +63,129 @@ paths:
               schema:
                 $ref: "#/components/schemas/LeonardoVersion"
       security: []
+
+## Runtime paths ##
+
+  /api/runtimes:
```

Comment:
should this be `api/google/v1/runtimes`? also, shall we rename the other path to `api/google/v1/runtimes` as well as opposed to singular form? I think by convention, the resource collection is usually plural.

Reply:
Yeah makes sense.. good catch